Wessel/FileChooser.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class FileChooser : MonoBehaviour {
+    public List<AudioClip> audioList = new List<AudioClip>();
+    public GameObject buttonPrefab;
+    public GameObject canvas;
+    private int position = -100;
+    private string path;
+
+
+    private void Start() {
+        path = Application.dataPath + "\\AudioFolder\\";
+        if (Application.platform == RuntimePlatform.WindowsPlayer) {
+            path = Application.dataPath;
+            path += "\\..\\";
+            path += "\\AudioFolder\\";
+        }
+        StartCoroutine(LoadAudioFile());
+    }
+
+    //this coroutine reads .wav files and imports them into the audio list
+    private IEnumerator LoadAudioFile()
+    {
```

Comment:
Line 27: Nog een haak op de volgende rij

Goed gebruik van single-line if statement. Voor de rest geen op- of aanmerkingen.

Reply:
Fixed!